<launch>
  <node name="warper" pkg="warper" type="warper" respawn="true" output="screen">
    <remap from="/usb_cam/image_raw" to="/cam0/image_raw"/>
    <remap from="/de_warped_image" to="/right/de_warped_image"/>
  </node>
  <node name="warper2" pkg="warper" type="warper" respawn="true" output="screen">
    <remap from="/usb_cam/image_raw" to="/cam1/image_raw"/>
    <remap from="/de_warped_image" to="/left/de_warped_image"/>
  </node>




</launch>

